src/scripts/components/LoginPage.tsx

```diff
@@ -53,12 +54,18 @@ export default function LoginPage ({ source, isLoggedIn, isPopup }: LoginProps)
     </Content>
   );
 
+  if (source === 'quickstart') {
+    document.title = "Welcome to Toggl Button!";
```

Comment:
I would extract this and all subsequent `document.title` calls  into a `React.useEffect`
to keep side-effects separate from render.